reports/hemang.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: report
+permalink: /GSoC/2017/ProjectReport/HemangKumar
+student: Hemang Kumar
+organisation: coala
+organisation_link : https://coala.io
+project: Improve coala Website & Supporting Tools
+project_link: https://summerofcode.withgoogle.com/projects/#6012382913495040
+tarball: https://github.com/hemangsk/GSoCReport/raw/master/hemangsk_coala.tar
+mentors: >
+ [Lasse Schuirmann](https://github.com/sils) & [Fabian Neuschmidt](https://github.com/fneu)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC-2017/milestones/3
+ - Phase 1 : https://gitlab.com/coala/GSoC-2017/milestones/7
+ - Phase 2 : https://gitlab.com/coala/GSoC-2017/milestones/8
+ - Phase 3 : https://gitlab.com/coala/GSoC-2017/milestones/9
+bio: >
+ I'm a final year student of Computer Science & Engineering at University School of Information,
+ Communication & Technology, GGS IP University. I participated in GSoC and worked with [coala](https://coala.io) to facilitate users to use coala online and functionality for one click Git Repository linting. I also worked on a bot to generate coafile, worked on improving UI of coala-html and configurability of coala-projects. I also worked on adding Notification and Thread support in the IGitt library.  
+social:
+ - GitHub:
+   - username: hemangsk
+   - link: https://github.com/hemangsk
+ - GitLab:
+   - username: hemangsk
+   - link: https://gitlab.com/hemangsk
+ - Gitter:
+   - username: hemangsk
+   - link: https://gitter.im/hemangsk
+ - HackerRank:
+   - username: hemangsk
+   - link: https://hackerrank/hemang
+email: hemangsk@gmail.com
+blog: http://hemangsk.github.io/stories.html
+activity:
+ - 0:
+   - repo: coala-quickstart 
+   - link: https://github.com/coala/coala-quickstart/commits/master?author=hemangsk
+   - details: > 
+      Add --allow-incomplete-sections mode and bump coala version.
+ - 1:
+   - repo: coalaCSS
+   - link: https://github.com/coala/coalaCSS/commits/master?author=hemangsk
+   - details: >
+      Create a CSS library to have consistency throughout coala web projects.
+ - 2:
+   - repo: coala Projects
+   - link: https://github.com/coala/projects/commits/master?author=hemangsk
+   - details: > 
+       Commits **f26ae52** through **6497def** were part of GSoC.
+ - 3:
+   - repo: IGitt
+   - link: https://gitlab.com/gitmate/open-source/IGitt/commit/718603aa2294a55206cfaa8806aa624791757ec4
+   - details: Add put method
+ - 4:
+   - repo: IGitt
```

Comment:
Did you have to mention IGitt commits separately because gitlab doesn't allow filter by author? Or is there any other reason. (Not that it has anything to do with the PR)

Reply:
Yeah thats the reason, no filter by author in GitLab